src/tools/ColorTool/ColorTool/ConsoleTargets/TerminalSchemeConsoleTarget.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ColorTool.ConsoleTargets
+{
+    class TerminalSchemeConsoleTarget : IConsoleTarget
+    {
+        public void ApplyColorScheme(ColorScheme colorScheme, bool quietMode)
+        {
+            Console.WriteLine("Copy and paste the following text into the schemes array of your Windows Terminal profiles.json file to add this color scheme. (Don't forget to add a comma separator after the previous scheme.)");
+            Console.WriteLine("{");
+            Console.WriteLine($"    \"name\": \"{colorScheme.Name}\",");
+
+            string[] colors =
+            {
+                "black",
+                "blue",
+                "green",
+                "cyan",
+                "red",
+                "purple",
+                "yellow",
+                "white",
+                "brightBlack",
+                "brightBlue",
+                "brightGreen",
+                "brightCyan",
+                "brightRed",
+                "brightPurple",
+                "brightYellow",
+                "brightWhite",
+            };
+            Action<uint, string, bool> writeColorItem = (uint color, string name, bool last) =>
+            {
+                uint r = color & (0x000000ff);
+                uint g = (color & (0x0000ff00)) >> 8;
+                uint b = (color & (0x00ff0000)) >> 16;
+                Console.WriteLine($"    \"{name}\": \"#{r.ToString("X2")}{g.ToString("X2")}{b.ToString("X2")}\"{(last ? "" : ",")}");
```

Comment:
```suggestion
                Console.WriteLine($"    \"{name}\": \"#{r:X2}{g:X2}{b:X2}\"{(last ? "" : ",")}");
```

I am relatively sure that doing the whole `.ToString()` business is unnecessary with [interpolated strings](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated) and it can be handled this way.